tests/SpecTests/Operation.php

```diff
@@ -838,7 +827,6 @@ private function getResultAssertionTypeForDatabase()
     /**
      * Prepares a request element for a bulkWrite operation.
      *
-     * @param stdClass $request
      * @return array
```

Comment:
`@return array` is redundant. This may have just been missed by tooling, as I see it left behind for a lot of methods that already have an `array` return type hint.

Reply:
Removed this and a number of other superfluous type annotations.